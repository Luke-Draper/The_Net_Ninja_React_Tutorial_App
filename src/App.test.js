import React from "react";
import App from "./App";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("App renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<App />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
