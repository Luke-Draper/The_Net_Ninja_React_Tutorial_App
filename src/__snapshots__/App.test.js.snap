// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  className="App"
>
  <nav
    className="nav-wrapper grey darken-3"
  >
    <div
      className="container"
    >
      <a
        className="brand-logo"
        href="/"
        onClick={[Function]}
      >
        MarioPlan
      </a>
      <ul
        className="right"
      >
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            New Project
          </a>
        </li>
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            Log Out
          </a>
        </li>
        <li>
          <a
            aria-current="page"
            className="btn btn-floating red lighten-1 active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            LD
          </a>
        </li>
      </ul>
      <ul
        className="right"
      >
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            Sign Up
          </a>
        </li>
        <li>
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            Log In
          </a>
        </li>
      </ul>
    </div>
  </nav>
  <div
    className="dashboard container"
  >
    <div
      className="row"
    >
      <div
        className="col s12 m6"
      >
        <div
          className="project-list section"
        >
          <div
            className="card z-depth-0 project-summary"
          >
            <div
              className="card-content grey-text text-darken-3"
            >
              <span
                className="card-title"
              >
                Project Title
              </span>
              <p>
                Posted by Luke Draper
              </p>
              <p
                className="grey-text"
              >
                3rd September, 2018
              </p>
            </div>
          </div>
          <div
            className="card z-depth-0 project-summary"
          >
            <div
              className="card-content grey-text text-darken-3"
            >
              <span
                className="card-title"
              >
                Project Title
              </span>
              <p>
                Posted by Luke Draper
              </p>
              <p
                className="grey-text"
              >
                3rd September, 2018
              </p>
            </div>
          </div>
          <div
            className="card z-depth-0 project-summary"
          >
            <div
              className="card-content grey-text text-darken-3"
            >
              <span
                className="card-title"
              >
                Project Title
              </span>
              <p>
                Posted by Luke Draper
              </p>
              <p
                className="grey-text"
              >
                3rd September, 2018
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col s12 m5 offset-m1"
      >
        <div>
          <p>
            Notifications
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
